reference_guide/api_changes/api_changes_list_2020.md

```diff
@@ -210,7 +215,7 @@ Java EE plugins split
 ### Changes in PhpStorm and PHP Plugin 2020.2
 
 Added Union Types Support
-: Please see [PhpStorm Breaking Changes](/products/phpstorm/php_open_api_breaking_changes.md).
+: See [Breaking Changes in PhpStorm 2020.2](/products/phpstorm/php_open_api_breaking_changes_2002.md).
```

Comment:
broken link /products/phpstorm/php_open_api_breaking_changes_2003.md

Reply:
Missed that, thanks!